### PR TITLE
HydroHUDs can scan fruits / seeds.

### PR DESCRIPTION
**Pull request by gurfan**
## Why it's good
Allows botanists to completely exchange the hydro analyzer for the HydroHUD.

## Changelog
:cl:
 * rscadd: HydroHUDs can now scan fruits and seeds by examining them.

### CODE REVIEW

File: code/modules/hydroponics/seeds.dm [hunk 1]
```diff
@@ -44,6 +44,9 @@
 	else
 		to_chat(user, "Plant Yield: <span class='info'>[(seed.yield != -1) ? seed.yield : "<span class='warning'> ERROR</span>"]</span>")
 		to_chat(user, "Plant Potency: <span class='info'>[(seed.potency != -1) ? seed.potency : "<span class='warning'> ERROR</span>"]</span>")
+	var/mob/living/carbon/human/H = user
+	if(istype(H) && H.is_wearing_item(/obj/item/clothing/glasses/hud/hydro))
```
Comment (Kurfursten) [inline code/modules/hydroponics/seeds.dm]: might as well just make the hydrovision proc a global one or even a define

**Comment by Kurfursten** [comment]
Just fix it, this is a tiny change!

**Comment by gurfan** [comment]
fixed

**Comment by gurfan** [comment]
fixed (real)

**Pull request merged**